motr/m0crate/parser.c

```diff
@@ -442,6 +433,32 @@ int copy_value(struct workload *load, int max_workload, int *index,
 			else
 				parser_emit_error("Unkown key ordering: '%s'", value);
 			break;
+		case KEY_SIZE:
+			w = &load[*index];
+			ciw = workload_index(w);
+			if (!strcmp(value, "random"))
```

Comment:
```
int strcmp(const char *s1, const char *s2);
```
According to "doc/coding-style",
this code should be written as:
```
if (strcmp(value, "random") == 0)
```

Reply:
Fixed.